grantnav/frontend/views.py

```diff
@@ -303,7 +306,7 @@ def get_clear_all(request, context, json_query):
     except KeyError:
         json_query["query"]["bool"]["filter"] = copy.deepcopy(BASIC_FILTER)
         current_filter = json_query["query"]["bool"]["filter"]
-        
```

Comment:
I have a filter in Sublime Text that strips trailing whitespace, hence this stuff in my PR